src/modules/BlobAnalysis/Module.js

```diff
@@ -0,0 +1,47 @@
+
+module.exports = function BlobAnalysis(options, UI){
+
+  var output;
+  
+  function draw(input, callback, progressObj) {
+  
+    progressObj.stop(true);
+    progressObj.overrideFlag = true;
+  
+    var step = this;
+    if(!options.step.inBrowser) {
+      this.output = input;
```

Comment:
@aashna27 We can change this! Since we now have modules running inside puppeteer! You can take a look at webgl-distort to see how to do that.